docs/overview.md

```diff
@@ -18,8 +18,8 @@ Most models consist of one class to represent the model itself; one class (or mo
 
 - `mesa.Model`
 - `mesa.Agent`
-- [mesa.time](apis/time.md)
-- [mesa.space](apis/space.md)
+- [mesa.time](https://mesa.readthedocs.io/en/stable/apis/time.html)
```

Comment:
The branch name (stable) can't be hardcoded, because with this, new changes from the PRs can no longer be checked.

Reply:
You have to figure out RTD and Sphinx configuration such that `apis/space.md` gets translated to the correct URL path upon build.

Reply:
Oh yes, I thought it needs to be configured using the folder structure rather than hardcoding the paths, because it looked like auto-documentation generator tool. Will look into it, and try to solve the issue. Thanks!